x-pack/plugins/security_solution/public/common/components/event_details/reason.tsx

```diff
@@ -44,47 +29,11 @@ export const ReasonComponent: React.FC<Props> = ({ eventId, data }) => {
     return aadReason.length > 0 ? aadReason : siemSignalsReason;
   }, [data]);
 
-  const ruleId = useMemo(() => {
-    const siemSignalsRuleId = getFieldValue({ category: 'signal', field: 'signal.rule.id' }, data);
-    const aadRuleId = getFieldValue({ category: 'kibana', field: ALERT_RULE_UUID }, data);
-    return aadRuleId.length > 0 ? aadRuleId : siemSignalsRuleId;
-  }, [data]);
-
   if (!eventId) {
     return <EuiTextColor color="subdued">{EVENT_DETAILS_PLACEHOLDER}</EuiTextColor>;
   }
 
-  return reason ? (
```

Comment:
The link to the rule now lives in one of the cards in the overview, so we could remove it from here.